### ISSUE TITLE
please add an option to disable tax inclusion

### ISSUE THREAD

**Issue by LastExceed** [fmauNeko/MarketBoardPlugin#17]
would be very useful for undercutting others

**Comment by kgetit** [comment]
This would be a great option.

**Comment by Amaria77** [comment]
Agreed. That's what I downloaded it for and alas it's not particularly helpful in that regard due to the tax inclusion.

**Comment by EnderSyth** [comment]
Agreed, I would very much appreciate the option to toggle it, as its more useful to match the price of the market board exactly excluding tax for both sellers (obviously for undercutting) and for buyers as the tax rate is both flat (so always present at the same level) and not listed in the in game market board.
